modules/deepvariant/main.nf

```diff
@@ -0,0 +1,46 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DEEPVARIANT {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::deepvariant=1.2.0" : null)
```

Comment:
(New to nextflow/nf-core so I could be totally wrong here) Doesn't deepvariant on bioconda lack the run_deepvariant executable? If so, wouldn't the script below fail should someone chooses to run the bioconda version?

Reply:
Hi @ramprasadn , yes that's true.

> If so, wouldn't the script below fail should someone run the bioconda version of DV?

However, I'm not sure if the Google team officially provides any pip / conda package for this https://github.com/google/deepvariant#official-solutions 

So, for the time being I've left this snippet here. Good notice though 👍

Reply:
What's the point of the deepvariant package on bioconda then? 

Anyway, in that case conda is simply not supported for this module (we have this for others, too, e.g. cellranger. 
Since there is no dedicated singularity image, the check for `params.singularity_pull_docker_container` is not necessary either. 

Check out how this is solved in the cellranger module: 
https://github.com/nf-core/modules/blob/bf5edb86148d881965c62eafef08c09a5a4a62ba/modules/cellranger/count/main.nf#L5-L8

Reply:
I've incorporated this suggestion with https://github.com/nf-core/modules/pull/572/commits/9a87d414a16c9158eb70c3ce3d2518fef3b257d1 - thanks @grst !